CredentialProvider.Microsoft/Util/HttpClientFactory.cs

```diff
@@ -21,15 +22,12 @@ public HttpClientFactory(HttpClient httpClient) : base(httpClient)
 
         static HttpClientFactory()
         {
-            // https://learn.microsoft.com/en-us/dotnet/fundamentals/networking/http/httpclient-guidelines
-#if NETFRAMEWORK
-            var httpClient = new HttpClient();
-#else
-            var httpClient  = new HttpClient(new SocketsHttpHandler
+            var httpClient = new HttpClient(new HttpClientHandler
```

Comment:
Your default HTTP client factory is now pretty much identical to MSAL's. Might as well not provide one. 

https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/blob/main/src/client/Microsoft.Identity.Client/PlatformsCommon/Shared/SimpleHttpClientFactory.cs

Reply:
huh you are right. @JohnSchmeichel  I thought we were doing this for user agent info?

Reply:
ah the magic is in the base class `MsalHttpClientFactory`